test/functional/tests/fault_injection/test_soft_hot_plug_device.py

```diff
@@ -50,6 +50,8 @@ def test_soft_hot_plug_cache(cache_mode):
 
     with TestRun.step("Soft hot unplug cache device"):
         cache_dev.unplug()
+        if cache_dev.is_detected():
+            TestRun.block("Failed to unplug the device!")
```

Comment:
This check should be performed by the TF itself.

Reply:
Actually it is:
![image](https://user-images.githubusercontent.com/25607940/123778103-1199f900-d8d1-11eb-85c3-04582d3ec526.png)
But if invalid serial has been set for a disk, TF reports the disk as already unplugged. After reconsidering the issue I will add a check to the function responsible for parsing configuration file.